packages/cms/src/components/topics/Card.jsx

```diff
@@ -15,18 +15,27 @@ export default class Card extends Component {
     const selectors = contents.selectors || [];
 
     const statGroups = nest().key(d => d.title).entries(stats);
+    const statGroupsMulti = nest().key(d => d.title).entries(stats);
+    statGroupsMulti.shift();
 
-    return <div className={ `topic pt-card pt-elevation-0 ${slug} Card` } ref={ comp => this.topic = comp }>
+    return <div className={ `topic bp3-card bp3-elevation-0 ${slug} Card` }>
```

Comment:
These `bp3` classes should stay as `pt`. You may have noticed an issue if you had installed canon-core v0.17, but cms v0.1 is locked to v0.16 of canon-core, which uses the `pt` prefixed.